lib/list.js

```diff
@@ -49,18 +56,22 @@ module.exports = class List {
 }
 
 function cas (prev, next) {
-  return !same(prev.value, next.value, { strict: true })
+  if (prev.value.type !== next.value.type) return true
+  if (prev.value.description !== next.value.description) return true
+  if (prev.value.seeders !== next.value.seeders) return true
+  return false
 }
 
 const keyEncoding = {
   preencode (state, k) {
-    c.string.fixed(52).preencode(state, k)
+    c.fixed32.preencode(state, k)
   },
   encode (state, k) {
-    c.string.fixed(52).encode(state, k)
+    c.fixed32.encode(state, k)
   },
   decode (state) {
-    return c.string.fixed(52).decode(state)
+    const key = c.fixed32.decode(state)
+    return Id.encode(key)
```

Comment:
I made it work directly with the buffer, but it unncessarily complicates when working with the UI, like too much Id encode/decode

This was the middleground, key saved it in raw but used as string

Reply:
that shouldnt be done here, an encoding should be "sound", ie encode(decode()) should work

Reply:
if you wanna use string keys, just use c.string and dont fix them in size as thats not spec'ed

Reply:
thats perfectly fine